public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardForm.tsx

```diff
@@ -85,7 +89,15 @@ export const SaveDashboardForm = ({
             />
           )}
 
-          <TextArea {...register('message')} placeholder="Add a note to describe your changes." autoFocus rows={5} />
+          <TextArea
+            {...register('message')}
+            aria-label="message"
+            value={message}
+            onChange={onUpdateMessage}
```

Comment:
This will override `onChange` handler provided by the `register` function. What you need to do instead is:

```ts
const messageProps = register('message');
...

onChange={(e) => {
  // your update here 
  ...
  // original callback call
  messageProps.onChange(e);
}}
```

Reply:
I tried to fix it, but it would be much better to update `react-hook-form`(https://github.com/react-hook-form/react-hook-form/releases/tag/v7.16.0) to be able to handle such cases